prime-router/assets/csv-download-site/index__inline.html

```diff
@@ -18,10 +18,16 @@
             }
     
             function logout(){
-                removeItem("user")
-                removeItem("organization")
+                removeItem("jwt")
                 window.location.replace("http://www.cdc.gov");
             }
```

Comment:
I thought we going to logout to the site?

Reply:
we do - thats the window.location.replace() call...